src/csync/csync_reconcile.cpp

```diff
@@ -463,43 +463,21 @@ static void _csync_merge_algorithm_visitor(csync_file_stat_t *cur, CSYNC * ctx)
     //hide instruction NONE messages when log level is set to debug,
     //only show these messages on log level trace
     const char *repo = ctx->current == REMOTE_REPLICA ? "server" : "client";
-    if(cur->instruction ==CSYNC_INSTRUCTION_NONE)
+    if(cur->type == ItemTypeDirectory)
```

Comment:
How do you know they're not useful in an arbitrary future situation?

Reply:
I thought about it and made adjustments. Since each file is now mentioned once during update phase (that was added earlier) it's indeed unnecessary to talk about NONE files during reconcile. I also removed some redundant lines from the update phase again.